RadarCovid/Application/JailBreakDetect.swift

```diff
@@ -24,7 +24,8 @@ class JailBreakDetect {
         #if arch(i386) || arch(x86_64)
             // This is a Simulator not an idevice
             return false
-        #endif
+
```

Comment:
as a suggestion, you can extract the macros to an external descriptive method:
```
static var isSimulator: Bool {
        #if arch(i386) || arch(x86_64)
            return true
        #else
            return false
        #endif
    }
```
On this way, you reduce 1-level of indentation and let code more descriptive using a `guard !isSimulator else { return false }` for example.

Reply:
The code should not rely on the current architecture to check if we're on the simulator or not because the simulator will run on ARM architecture on the future Apple Silicon macs and the current mac-catalyst platform is x86_64.

So, to check if we're on a simulator or not, the proper solution is `#if targetEnvironment(simulator)` wherever this condition is required. This conditionally builds simulator or actual device code and does not relegate the check until run time.

Reply:
Fixed in 6e03017da35a4405c358e9388bb378b80e169a7d